swatch-contracts/ct/pom.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.redhat.swatch</groupId>
+    <artifactId>swatch-quarkus-parent</artifactId>
+    <version>1.1.0-SNAPSHOT</version>
+    <relativePath>../../swatch-quarkus-parent/pom.xml</relativePath>
+  </parent>
+
+  <artifactId>swatch-contracts-component-tests</artifactId>
+  <name>SWATCH - Services - Contracts - Component Tests</name>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.redhat.swatch</groupId>
+      <artifactId>swatch-core</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.redhat.swatch</groupId>
+      <artifactId>swatch-test-framework</artifactId>
+    </dependency>
+    <!-- Needed for static imports of Channels and any API models used by CTs -->
+    <dependency>
+      <groupId>com.redhat.swatch</groupId>
+      <artifactId>swatch-contracts</artifactId>
+      <version>${project.version}</version>
```

Comment:
This implies having to build many other dependencies which will slowdown the component tests overall execution. 
In this case, it's better to duplicate what is needed, or create another common dependency.